ui/application-window.ui

```diff
@@ -38,6 +38,51 @@ Author: Philipp Wolfer <ph.wolfer@gmail.com>
     <property name="can_focus">False</property>
     <property name="icon_name">media-playback-stop-symbolic</property>
   </object>
+  <object class="GtkPopover" id="pop_menu">
+    <property name="can_focus">False</property>
+    <child>
+      <object class="GtkBox">
+        <property name="visible">True</property>
+        <property name="can_focus">False</property>
+        <property name="orientation">vertical</property>
+        <property name="margin">6</property>
```

Comment:
The proper margin for the primary menu is 12, not 6 - that's a mistake I made in the Music implementation.